views/playground/workflow-designer/index.html.twig

```diff
@@ -0,0 +1,57 @@
+{% extends '@pulsar/lexicon/base.html.twig' %}
+
+{% import '@pulsar/pulsar/v2/helpers/html.html.twig' as html %}
+{% import '@pulsar/pulsar/v2/helpers/form.html.twig' as form %}
+{% import '@pulsar/pulsar/v2/helpers/tabs.html.twig' as tabs %}
+{% import '@pulsar/pulsar/v2/helpers/flash.html.twig' as flash %}
+
+{% block title %}{{ parent() }}{% endblock %}
+
+<<<<<<< HEAD:views/playground/workflow-designer/index.html.twig
```

Comment:
Bad merge

Reply:
Highly recommend not resolving merge conflicts by hand and rather using something like PhpStorms conflict tool. A code review should be your last line of defence, not your first

Reply:
It's a PR having other PRs cherry picked into it, merge conflicts are inevitable. Chill dude.